week4/w4t1.py

```diff
@@ -0,0 +1,10 @@
+a = []
+k = 0
+n = int(input())
+for i in range(n):
+    c = int(input())
+    a.append(c)
```

Comment:
```suggestion
    a.append(int(input()))
```
конечно можно вынести это в переменную и дать ей имя для повышения читаемости кода, но тогда это имя дожно быть не `c`